script

```diff
@@ -2,8 +2,9 @@
 while :
     do
         now=$(date +"%Y%m%d%H%M%S")
-        /opt/encoded/checkfiles/bin/python /opt/encoded/checkfiles/checkfiles.py "$@" --out $now-checkfiles.log --err $now-checkfiles-error.log --include-unexpired-upload 2> $now-file-errors.log 1> $now-file-output.log
-        /opt/encoded/checkfiles/bin/python /opt/encoded/checkfiles/checkexperiments.py "$@" --out $now-checkexperiments.log --err $now-checkexperiments-error.log 2> $now-exp-errors.log 1> $now-exp-output.log
-        /opt/encoded/checkfiles/bin/python /opt/encoded/checkfiles/checkmd5.py "$@" --out $now-checkmd5.log 2> $now-md5-errors.log 1> $now-md5-output.log
+        url=$(echo "$@" | rev | cut -d " " -f 1 | rev | sed -e 's|^[^/]*//||' -e 's|/.*$||' | cut -d "." -f 1)
```

Comment:
This line is pretty beastly, especially the `sed`. Could you add a comment explaining what it does?

Reply:
good idea. I did this awhile and forgot what this even did too. I ended up finding an unnecessary sed there. So I went ahead and revised this by making this url parsing into two lines with comments.